### PR TITLE
[GOVCMSD9 713] add security.txt module to the drupal contrib module list

### PR DESCRIPTION
**Pull request by rsau**

### CODE REVIEW

File: govcms.info.yml [hunk 1]
```diff
@@ -28,6 +28,7 @@ install:
   - page_cache
   - path
   - rdf
+  - securitytxt
```
Comment (mingsong-hu) [inline govcms.info.yml]: Can a site admin uninstall this module?

File: composer.json [hunk 2]
```diff
@@ -126,6 +126,7 @@
         "drupal/search_api_attachments": "9.0.0",
         "drupal/search_api_solr": "4.2.7",
         "drupal/seckit": "2.0.0",
+        "drupal/securitytxt": "^1.2",
```
Comment (mingsong-hu) [inline composer.json]: we have to specify the exactly version to 1.2.0

**Comment by mingsong-hu** [comment]
For existing GovCMS sites, we need to create a update hook in the GovCMS Security module to install this module.

File: composer.json (hunk 2 above)
Comment (rsau) [reply composer.json]: Done

**Review by ruwanl — APPROVED** [review]

**Pull request merged**